README_dependencies.md

```diff
@@ -29,18 +29,23 @@ RefreshEnv.cmd # reload the environment
 ```
 
 ### Necessary Dependencies
-1. A C++ compiler that supports C++17.
+1. A C++ compiler that supports C++20.
 See [cppreference.com](https://en.cppreference.com/w/cpp/compiler_support)
 to see which features are supported by each compiler.
 The following compilers should work:
 
-  * [gcc 7+](https://gcc.gnu.org/)
+  * [gcc 11+](https://gcc.gnu.org/)
 	<details>
 	<summary>Install command</summary>
 
-	- Debian/Ubuntu:
+	- Ubuntu 21.04+:
+
+			sudo apt install gcc-11
+
+	- Ubuntu 18.04-20.04:
 
-			sudo apt install build-essential
+			sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && sudo apt-get update -qq
+			sudo apt install gcc-11
```

Comment:
This probably deserves comment. I am removing instructions on how to install gcc-11 on Debian. 

I've looked into installing gcc-11 on Debian briefly, and it looks complicated enough that I would have to manually test the instructions I've found. I'm not going to do that, because Debian not widely used. I think it would make more sense to add instructions for Arch-derived distros and Fedora.